contrib/completion/bash/sdk

```diff
@@ -25,21 +25,19 @@ __sdkman_complete_command() {
 		sdk)
 			candidates=("install" "uninstall" "list" "use" "config" "default" "home" "env" "current" "upgrade" "version" "broadcast" "help" "offline" "selfupdate" "update" "flush")
 			;;
-		current|default|home|uninstall|upgrade|use)
+		current|c|default|d|home|h|uninstall|rm|upgrade|ug|use|u)
 			local -r candidate_paths=("${SDKMAN_CANDIDATES_DIR}"/*)
 
 			for candidate_path in "${candidate_paths[@]}"; do
 				candidates+=(${candidate_path##*/})				
 			done
 			;;
-		install|list)
-			local -r all_candidates=$(curl --silent "${SDKMAN_CANDIDATES_API}/candidates/all")
-
+		install|i|list|ls)
 			while IFS= read -d, -r candidate; do
 				candidates+=($candidate)
-			done <<< "$all_candidates"
+			done <<< "$(curl --silent "${SDKMAN_CANDIDATES_API}/candidates/all")"
```

Comment:
As in your question in Slack, it is better to use the cached candidates. Not only is it more performant, but it mitigates load on the candidates API.